docs/kittengrad.md

```diff
@@ -139,13 +139,81 @@ of duplicate inserts without a replay. Updates however are dependent
 on the previous chain pointer, which requires a new datastore write
 and replay of the transaction.
 
-### Steady State Operation
+### Block Commit Protocol
 
-TBD
+The objective of the block commit protocol is to establish consensus
+among transactors for the next block. In the long term, we want to
+utilize a PoW or PoS system that is robust to byzantine failures
+(eg tendermint). At this stage of development however, there is nothing
+at stake; we simply want a simple and robust protocol that can scale to some
+100MM artefacts. 
+
+So we treat the Block Commit Protocol as a pluggable module and we call
+the baseline implementation BCP0.
+
+The BCP0 protocol operates in rounds, with each round having three phases:
+1. A Proposal phase, where one or more new blocks are proposed, validated and propagated by transactors
+2. A voting phase, where transactors vote for the next block
+3. A commit phase, where a block is commited.
+
+#### Block Proposal
+
+During the block proposal phase, one or more transactors with stake in the
+form of pending transactios,  generate a block and brodcast a signed block proposal
```

Comment:
pending transactions = transactions submitted *directly* to that node, not rebroadcast txes, right?